service/src/main/resources/HTMLTemplate.html

```diff
@@ -1376,9 +1385,8 @@ <h3 class="heading heading--two">Applicant</h3>
     <div class="data-list__row">
       <dt class="data-list__title">Contact name</dt>
       <dd class="data-list__definition">
-        {% if case_data.applicant is not empty and
-        case_data.applicant.personToContact is not empty %}
-          {{ case_data.applicant.personToContact }}
+        {% if applicant.value.party.telephoneNumber is not empty %}
+          {{ applicant.value.party.telephoneNumber.contactDirection }}
```

Comment:
What if contact direction is empty? it will display nothing instead of displaying `-`. Condition seems wrong.

Reply:
👍